.travis.yml

```diff
@@ -68,6 +68,10 @@ matrix:
       dist: xenial
       env: TOXENV=py37
       <<: *not-on-master
+    - python: "3.8-dev"
+      dist: xenial
```

Comment:
Xenial environment is the default one for Travis: https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment

So `dist: xenial` is not needed anymore.